tests/unit/test_progression_manager.py

```
@@ -0,0 +1,130 @@
+"""
+Unit tests for the ProgressionManager class.
+"""
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)